packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -359,21 +333,23 @@ export namespace WorkspaceFrontendContribution {
             };
         }
 
-        // In electron, only workspace files can be selected when the multi-root workspace feature is enabled.
-        if (supportMultiRootWorkspace) {
+        // If electron
+        if (OS.Type.OSX === type) {
+            // `Finder` can select folders and files at the same time. We allow folders and workspace files.
             return {
                 title,
                 canSelectFiles: true,
-                canSelectFolders: false,
+                canSelectFolders: true,
                 filters: DEFAULT_FILE_FILTER
             };
         }
 
-        // Otherwise, it is always a folder.
+        // In electron, only workspace files can be selected when the multi-root workspace feature is enabled.
```

Comment:
is mentioning "multi-root" here is necessary ? since we remove the flag

Reply:
the flag will come back. so i will keep it as is.